examples/platforms/nrf52840/usb-cdc-uart.c

```diff
@@ -123,6 +124,7 @@ static void cdcAcmUserEventHandler(app_usbd_class_inst_t const *aCdcAcmInstance,
     default:
         break;
     }
+    otSysEventSignalPending();
```

Comment:
nit: please add a new line before.

Theoretically, it would be enough to make this call inside `APP_USBD_CDC_ACM_USER_EVT_TX_DONE` and `APP_USBD_CDC_ACM_USER_EVT_RX_DONE`, but i agree that for simplification it is okay to keep it as it is.